src/dynamics/video_player/player/states.js

```diff
@@ -656,14 +656,21 @@ Scoped.define("module:VideoPlayer.Dynamics.PlayerStates.PlayOutstream", [
         dynamics: ["adscontrolbar"],
 
         _started: function() {
-            // if player is not hidden below method will do nothing
-            this.dyn.showHiddenPlayerContainer();
+            this.listenOn(this.dyn.channel("ads"), "impression", function() {
+                // if player is not hidden below method will do nothing
+                this.dyn.showHiddenPlayerContainer();
```

Comment:
Adding this since when we have multiple bad ads in a fallback, the player was showing up with a black screen with no ad content.

Reply:
I'd add your comment here into the code's comment.  This is a hack so context is needed